sdk/cpp/core/src/xml_subtree_codec.cpp

```diff
@@ -326,7 +308,13 @@ static void decode_xml(xmlDocPtr doc, xmlNodePtr root, Entity &entity,
       check_and_set_content(entity, leaf_name, xml_node->parent,
                             xml_node->content, doc);
     } else {
-      check_and_set_node(entity, parent, xml_node, doc);
+      // We skip the decode of "input"
```

Comment:
Need to explain about the restriction of forwarding every XML child to the `input` child entity.

Reply:
We need to add the following explanation:
the current implementation requires that if a top-level entity has an "input" child node it will be the only child, or at least the only child in the XML string that is being decoded.
This solution also prohibits us from using "input" as a name for a child of a non-RPC top-level entity.
also note that this solution still encodes the "input" node (i.e. we cannot decode the result of encode) and does not solve a similar issue for decoding an RPC's "output" child.